application/src/main/scala/com/azavea/franklin/tile/GeoTiffInfoReader.scala

```diff
@@ -0,0 +1,139 @@
+package com.azavea.franklin.tile
```

Comment:
Everything here is to be able to cache metadata and tiff info for faster data retrieval